src/sentry/release_health/duplex.py

```diff
@@ -729,16 +731,25 @@ def should_compare(val: ReleaseSessionsTimeBounds) -> bool:
         )
 
     def check_has_health_data(
-        self, projects_list: Sequence[ProjectOrRelease]
+        self,
+        projects_list: Sequence[ProjectOrRelease],
+        now: Optional[datetime] = None,
```

Comment:
Does the duplex impl actually need the `now` argument, or is it sufficient to add it to the sessions and metrics implementation? It's not part of the base class interface at `release_health.base`.

Reply:
i think we shouldn't change the signature in subclasses, i just forgot to update the baseclass and apparently mypy can't be bothered to warn me about it either 🙃